From: Wineforge engine builders
Subject: [PATCH] winhlp32: Remove unused bool field

Modern MinGW headers define bool as a type. The unused field therefore fails
to compile. CrossOver 25.1.1 removes it; backport that one-line change.

---
 programs/winhlp32/macro.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/programs/winhlp32/macro.h b/programs/winhlp32/macro.h
index 1111111..2222222 100644
--- a/programs/winhlp32/macro.h
+++ b/programs/winhlp32/macro.h
@@ -26,7 +26,6 @@ typedef BOOL (*WINHELP_LookupWord)(LPCSTR, unsigned, struct lexret*);
 
 struct lexret {
   LPCSTR        proto;
-  BOOL          bool;
   LONG          integer;
   LPCSTR        string;
   FARPROC       function;
-- 
2.50.1
